src/ubx.cpp

```diff
@@ -520,6 +520,8 @@ int GPSDriverUBX::configureDevice(const GNSSSystemsMask &gnssSystems)
 	cfgValset<uint8_t>(UBX_CFG_KEY_NAVSPG_UTCSTANDARD, 3 /* USNO (U.S. Naval Observatory derived from GPS) */,
 			   cfg_valset_msg_size);
 	cfgValset<uint8_t>(UBX_CFG_KEY_NAVSPG_DYNMODEL, _dyn_model, cfg_valset_msg_size);
+	// The rover will stop using RTCM corrections that are older than 2 seconds
+	cfgValset<uint8_t>(CFG_NAVSPG_CONSTR_DGNSSTO, 2, cfg_valset_msg_size);
```

Comment:
Probably ok, but it means it will also fall out of RTK with base station quite quickly

Reply:
From your experience, the RTCM data does not arrive at 1Hz? 

The other options that I see are: 

- set it to a higher value i.e. 3 
-  create a parameter for `CFG_NAVSPG_CONSTR_DGNSSTO` and let the user chose.

Reply:
> From your experience, the RTCM data does not arrive at 1Hz?

Yes but the datalink might not always have perfect connectivity.

We can set it to 3. The other option would be to just set it to the one that gets the heading corrections (it does not get the RTK base station corrections at the same time).